src/groups/bmq/bmqimp/bmqimp_queuemanager.h

```diff
@@ -363,8 +352,9 @@ class QueueManager {
     /// `correlationId`, and return a shared pointer to the Queue object (if
     /// found), or an empty shared pointer (if not found).
     QueueSp lookupQueueBySubscriptionId(bmqt::CorrelationId* correlationId,
-                                        int                  queueId,
-                                        unsigned int subscriptionId) const;
+                                        unsigned int* subscriptionHandle,
```

Comment:
We have `subscriptionHandle` in some places like here, and `subscriptionHandleId` in other places.